src/NetTopologySuite/IO/WKTReader.cs

```diff
@@ -865,7 +865,7 @@ private MultiPoint ReadMultiPointText(TokenStream tokens, GeometryFactory factor
             string nextToken = GetNextEmptyOrOpener(tokens);
             if (nextToken.Equals(WKTConstants.EMPTY))
             {
-                return factory.CreateMultiPoint(new Point[0]);
+                return factory.CreateMultiPoint(Array.Empty<Point>());
```

Comment:
:page_facing_up: Same basic note about behavior change as in `MinimumBoundingCircle.cs`